lib/logstash/outputs/elasticsearch.rb

```diff
@@ -184,6 +184,16 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
   # See https://hc.apache.org/httpcomponents-client-ga/httpclient/apidocs/org/apache/http/impl/conn/PoolingHttpClientConnectionManager.html#setValidateAfterInactivity(int)[these docs for more info]
   config :validate_after_inactivity, :validate => :number, :default => 10000
 
+  # The target size of an HTTP request in megabytes to send. Elasticsearch has a hard limit of 100MB
+  # after which it will reject the request. If the batch received by this plugin exceeds this amount
+  # it will be split up and sent in multiple requests. A request may exceed this size if there is a single
+  # document larger than it. You can limit an individuals document's size with max_doc_megabytes
+  config :target_bulk_megabytes, :validate => :number, :default => 75
```

Comment:
I greatly prefer base units for settings, so `bytes` instead of `megabytes`.

I'll send you a PR to add bytes to your branch.